webpack.config.js

```diff
@@ -36,16 +39,21 @@ function getLoadersConfig(env) {
 }
 
 function getPluginsConfig(env) {
+  if(!config.availableLanguages) {
+    throw "availableLanguages needs to be configured in the config file";
+  }
+  var languageExpression = new RegExp("^./(" + config.availableLanguages.join('|') + ")$");
```

Comment:
I think that is ok, as it is possible to only include the config files present during build time.